src/GRANULAR/gsm.h

```diff
@@ -0,0 +1,62 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifndef LMP_GSM_H_
```

Comment:
Include guards should always be `LMP_AAA_BBB_E` if the file name is `aaa_bbb.h`

What worries me is the use of the "gsm" initialism. While it is certainly convenient, it would be better to have something "longer" and more verbose/specific instead as that reduces the risk that there is an include file with the same name elsewhere. We already had some problems in the past with, e.g. "memory.h" and "atom.h" and a few other "obvious" names.

Since this is not something that a user will have to enter, it should not be a big problem to use something like GranSubModel or GranSubM as base class name and thus `gran_sub_model.h` or `gran_subm.h` as corresponding file name. I am open for alternate suggestions.

Reply:
Originally, Dan and I debated fully spelling out GranularSubmodel to avoid the acronym. But I much prefer your shortened suggestions as an in-between. I'm happy with any of your suggestions or GranSubMod. I can make the changes.

Reply:
I don't have a strong feeling about any of those. Either will work for me as either will address the concern.

Reply:
I just pushed changes which should include all of these changes. For the include guard I have something like:
`#ifndef LMP_GRAN_SUB_MOD_H_`
for gran_sub_mod.h. Is this the correct format? Your example ended in `_E_` but I wasn't sure whether or not that was a typo.

Reply:
Yes, this was a typo. It was supposed to be `LMP_AAA_BBB_H` (note no trailing underscore).

Sorry about being so anal retentive about such minor details, but having consistent names for these can be a massive time saver when we have to make bulk changes, as those can be automated for all conforming files. I have been through such processes a few times and finding and fixing the non-conforming files has always been a bit of a hassle. :-(

Reply:
Ah, I totally overlooked the trailing underscore. I'll prune those. It also looks like the styles aren't being found so I'll look at that again and patch it.

Thanks!

Reply:
> It also looks like the styles aren't being found so I'll look at that again and patch it.

Did you change `cmake/Modules/StyleHeaderUtils.cmake` and `src/Make.sh` as needed for the change in file names?

Reply:
Yep, I missed some changes to the cmake style utils file. Think it's fixed now.